### PR TITLE
Update README + Add VIM temporary files to .gitignore

### PR DESCRIPTION
**Pull request by TristanDamron**
Satisfies the acceptance criteria for #80 and #81

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -11,54 +11,55 @@ endif::[]
...
 == Viewing the Site Locally
 CAUTION: Please make sure you have all required dependencies before continuing.
 
 1. _Clone or download this repository to your machine._
 2. _Open a terminal window. Change your current directory to the cloned project directory._
-3. _Inside of your terminal, type the following command:_
+3. _Using a Bash terminal, type the following command:_
+   
+`
+sh build.sh
```
Comment (capsulecorplab) [inline README.adoc:43]: line isn't rendering as a code block. try 

```
[source]
----
sh build.sh
----
```

as per [callouts](https://gist.github.com/tracphil/87fff6ab2796fd855e62dd852eda502d#callouts) for GFA

File: README.adoc [hunk 2]
```diff
@@ -11,54 +11,55 @@ endif::[]
...
+`
+
+NOTE: If you receive a permissions error on *nix or macOS, prepend _sudo_ to the command. For example, type the following command:
+
+`
+sudo sh build.sh
```
Comment (capsulecorplab) [inline README.adoc:49]: same issue as line 43

File: README.adoc [hunk 3]
```diff
@@ -11,54 +11,55 @@ endif::[]
...
-ruby convert.rb
-----
+=== Viewing the site through a local web server
 
-NOTE: The convert.rb script will convert the .adoc files inside of the project directory into .html files that can be viewed in a web brower. You may see a few warnings appear in the terminal window after running this script. In most cases, it's fine to ignore them. If you have a concern,  https://github.com/LearnTeachCode/code-coffee-compendium/issues/new?assignees=&labels=&template=feature_request.md&title=[please submit an issue.]
+There may be situations where you want to view the site locally through a local web server. In this case, rather than running `build.sh` in a Bash terminal, you should run `serve.sh`. For example, type the following command in the project directory:
 
-Double-click on the index.html file that was generated in the project directory to view the site in a web browser.
+`
+sh serve.sh
```
Comment (capsulecorplab) [inline README.adoc:59]: same issue as line 43

**Review by capsulecorplab — CHANGES_REQUESTED** [review]

**Comment by TristanDamron** [comment]
@capsulecorplab Added another commit to address your concerns.

**Review by capsulecorplab — CHANGES_REQUESTED** [review]
Hm, still not rendering correctly https://github.com/LearnTeachCode/code-coffee-compendium/blob/b0afb5a3f9ffd1e10e396c67d6bb96a8b4073364/README.adoc

try adding another dash that's sandwiching the code snippet,
```
[source]
----
sh command
----
```

**Comment by TristanDamron** [comment]
@capsulecorplab Updated my PR to fix the code blocks and to add a step for updating submodules if viewing the site locally!

**Review by capsulecorplab — APPROVED** [review]
___
                                     //   \\
                                    ||=====||
                                     \\___//
                                      ./O
                                  ___/ //|\\
                                 / o    /}
                                (       /
                                \      /
                                |     (
                                |      \
                                )       \
                               /         \
                             /            )
                           /              |
                         //             / /
                       /       ___(    ,| \
                     /       /    \     |  \
                    (      /  /   /\     \  \
                    \\   /___ _-_//'|     |  |
                     \\_______-/     \     \  \
                                      \-_-_-_-_-

                        "Seal of Approval"

**Pull request merged**